util/Setup/Program.cs

```diff
@@ -185,17 +186,19 @@ private static void PrintEnvironment()
         Console.WriteLine("\n");
     }
 
-    private static void MigrateDatabase(int attempt = 1)
+    private static void PrepareAndMigrateDatabase()
     {
         var vaultConnectionString = Helpers.GetValueFromEnvFile("global",
             "globalSettings__sqlServer__connectionString");
-        var migrator = new DbMigrator(vaultConnectionString, null);
+        var migrator = new DbMigrator(vaultConnectionString);
 
-        var log = false;
+        var enableLogging = false;
```

Comment:
ℹ️ If we determine this is unnecessary I can kill off the whole parameter down the chain.